### DISCUSSION TITLE
How to exclude paths only when some query params are included

### DISCUSSION THREAD

**Discussion by afrancis-caregility** [zalando/logbook#1653] [Q&A] [answered]
I have the following request URL's 

/abc/def
/abc/def?time=169435354354
/abc/def/details 

How do I exclude just the second URL with the request parameter def?time=169....
So, far I have been only able to do this /abc/def** which excludes all the 3 URL's

**Comment by whiskeysierra** [comment, marked as answer]
You can write a custom predicate with any logic you like. There is no
default that matches your requirement and there probably won't ever be one,
because it's just too much of a rare corner case.

The custom predicate is meant to be an escape hatch for exactly those cases.

**Reply by afrancis-caregility** [reply]
Thanks.
